### PR TITLE
chore(main): Supporting full lifecycle for comments (update, delete)

### PR DESCRIPTION
**Pull request by maxdanilov**
Related: https://github.com/camunda/engineering-leadership-team/issues/33

Two additional parameters:
- `target_issue_footer_template` - a template string to be added to every issue copied to the target. `{{<link>}}` is replaced with the URL of the source issue. Does not serve any other purpose aside from decorative and visually aligning with the contents of the copied comments
- `target_comment_footer_template` - a template string to be added to every comment copied to the target. `{{<link>}}` is replaced with the URL to the source comment, `{{<author>}}` is replaced with the username of the original comment author.
  - the link to the source in the target is required to support a lifecycle of a comment (e.g. when a source is edited, it helps to find the corresponding target in the target repository, by having a link to the source)

Behavioural change:
- supporting payload events `edited` and `deleted` for the comments
  - tracking done with the help of the `target_comment_footer_template` ([example](https://github.com/test-org-maxdanilov2/issues-test/issues/3#issuecomment-1496352741))
- appending `target_issue_footer_template` to the body of the copied issues ([example](https://github.com/test-org-maxdanilov2/issues-test/issues/3#issue-1652105666))
- when a comment is copied/updated to the target, the source comment has a reaction 🚀  added to it 
- when an issue is copied/updated to the target, the source issue has a reaction 🚀  added to it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,31 +8,7 @@ Alternatively, you can set two separate environment variables GITHUB_TOKEN_SOURC
...
-  required: false
-  default: ""
-sync_repo_labels:
-  description: 'Whether to sync the labels from source to target repository'
-  required: false
-  default: "true"
-```
+See the list of supported options in `action.yml`
  
 ## Example
```
Comment (Langleu) [inline README.md:13]: not sure whether it makes sense to adjust the example to include the edited event for comments:
```
  issue_comment:
    types: [created, edited]
```

or to add a comment that one should enable it when run with `only_sync_main_issue: false`

Comment (Langleu) [reply README.md:13]: looking at the code, `deleted` as well.

File: index.ts [hunk 2]
```diff
@@ -96,6 +109,7 @@ if (syncRepoLabels) {
 
 const payload = require(process.env.GITHUB_EVENT_PATH as string)
 const number = (payload.issue || payload.pull_request || payload).number
+const action: string = payload.action
```
Comment (Langleu) [inline index.ts:112]: maybe you could refactor the remaining `payload.action` mentionings in line 237, 213, 169

**Review by Langleu — APPROVED** [review]
Some small comments here and there, but nothing functionally wrong.
We may want to invest in the future to switch to async/await, I feel the code may profit from it in terms of readability.

Tested: Comments edit / issue edit / comments delete
Generally backwards compatible.

File: README.md (hunk 1 above)
Comment (maxdanilov) [reply README.md:13]: Good idea, added the two options there.

File: index.ts (hunk 2 above)
Comment (maxdanilov) [reply index.ts:112]: Nice catch, refactored.

File: github.ts [hunk 3]
```diff
@@ -116,6 +144,47 @@ export class GitHub {
...
+
+    public deleteComment(commentId: number): Promise<any> {
+        return this.octokit
+            .request('DELETE /repos/{owner}/{repo}/issues/comments/{comment_id}', {
+                owner: this.owner,
+                repo: this.repo,
+                comment_id: commentId,
+            })
+            .then(response => {
+                console.log(`Updated comment ${commentId}`)
```
[the remark this answers is not in the document]
Comment (maxdanilov) [reply github.ts]: Good catch, amended the comment.

**Pull request merged**